src/sass/_products.scss

```diff
@@ -0,0 +1,259 @@
+.products {
+  min-width: 320px;
+  width: 100%vw;
+  height: 100%;
+  padding: 0 20px;
+  position: absolute;
+  margin-left: 50%;
+  transform: translateX(-50%);
+
+  &__header {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    transform: translateX(-50%);
+    margin-top: 98px;
+    margin-left: 50%;
+    width: 200px;
+    height: 36px;
+
+    font-family: $header-font;
+    font-style: normal;
+    font-weight: 'regular';
+    font-size: 34px;
+    line-height: 1.147;
+    text-align: center;
+    letter-spacing: 'standard';
+    text-transform: uppercase;
+
+    color: $red-1;
+  }
+
+  &__header::before {
+    display: flex;
+    justify-content: center;
+    content: '100% natural';
+    margin-bottom: 15px;
+    margin-top: 121px;
+
+    font-family: $header-font;
+    font-style: normal;
+    font-weight: 'regular';
+    font-size: 18px;
+    line-height: 1.167;
+    text-transform: uppercase;
+
+    color: #ffa5ba;
+  }
+}
+
+.products__list {
+  margin-top: 337px;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  list-style: none;
+  gap: 130px;
+}
+
+.products__overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 0;
+  width: 100%;
+  background-color: $grey;
+  border-radius: 24px;
+  overflow: auto;
+  z-index: 99;
+  text-align: center;
+  padding-top: 0;
+  transition: width 0.5s;
+  opacity: 0.85;
+}
+
+.overlayshow {
+  height: 100%;
+  padding-top: 100px;
+  transition: width 0.5s, padding-top 0.5s;
+}
+
+.products__button {
+  position: relative;
+  z-index: 100;
+}
+
+.products__card {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  min-width: 280px;
+  max-width: 320px;
+  margin-left: 50%;
+  transform: translateX(-50%);
+  width: 100%vw;
+  height: 485px;
+  background: $pink;
+  border-radius: 24px;
+  &::before {
```

Comment:
Osobiście te obrazki bym chyba jednak wstawił po prostu jako obrazki a nie jako before. W tym ukladzie chyba czytniki stron tego nie wyłapią?

Tutaj też zapytałbym Jakuba czy tak to zostawić